### ISSUE TITLE
Failure to build on openbsd: syscall.Stat_t does has no field or method Mtimespec

### ISSUE THREAD

**Issue by phinze** [fsouza/go-dockerclient#296]
Hi there!

Just got this while trying to cross compile a Terraform release:

```
2 errors occurred:
--> openbsd/386 error: exit status 2
Stderr: # github.com/fsouza/go-dockerclient/vendor/github.com/docker/docker/pkg/system
../../fsouza/go-dockerclient/vendor/github.com/docker/docker/pkg/system/stat_unsupported.go:16: s.Mtimespec undefined (type *syscall.Stat_t has no field or method Mtimespec)

--> openbsd/amd64 error: exit status 2
Stderr: # github.com/fsouza/go-dockerclient/vendor/github.com/docker/docker/pkg/system
../../fsouza/go-dockerclient/vendor/github.com/docker/docker/pkg/system/stat_unsupported.go:16: s.Mtimespec undefined (type *syscall.Stat_t has no field or method Mtimespec)
```

I'm unable to dig in any further at the moment, but I wanted to make sure I got it reported right away. :+1:

**Comment by ewindisch** [comment]
:+1: also affects Nacl builds. This might be better filed in docker/docker?

:dibs:

**Comment by fsouza** [comment]
I think so. If they don't mind not building on openbsd or nacl, then we can copy their code or something like that.

**Comment by jen20** [comment]
This is fixed by docker/docker#21325  - I'll open up a PR here to update the vendoring.

**Comment by fsouza** [comment]
Closing too, we no longer vendor the dependencies (as libraries should not do).
